### PR TITLE
Downgrade react to 0.14.8

### PR DESCRIPTION
**Pull request by joofsh**
- Fixes dependency react-on-rails requiring 0.14.8

React v0.14.8 was released march 2016, so not an old version by any means

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -44,7 +44,7 @@
     "eslint-plugin-react": "^6.7.1",
     "mocha": "^3.1.2",
     "npm-shrinkwrap": "^6.0.2",
-    "react-addons-test-utils": "^15.4.0"
+    "react-addons-test-utils": "^0.14.8"
```
Comment (joofsh) [inline client/package.json]: The key was to make all 3 versions match `0.14.8`, which is what react-on-rails requires for now

**Comment by amprokop** [comment]
image

**Comment by amprokop** [comment]
tested this

**Comment by amprokop** [comment]
LGTM, build works for me

**Pull request merged**